Pipfile

```diff
@@ -7,28 +7,29 @@ verify_ssl = true
 bandit = ">=1.7.5"
 black = "==23.11.0"
 pylint = "==3.0.2"
-tomli = "==2.2.1"  # Needed for black on Python < 3.11
+tomli = "==2.2.1"
 
 [packages]
 aiohttp = "==3.13.2"
-async-timeout = {version = "==5.0.1", markers = "python_version < '3.11'"}  # Required by aiohttp
```

Comment:
I feel like we should not remove the comments explaining why it is included? Is there anything to support this idea?

Reply:
No reason, no. But since other packages also do not have those, i want to put everything on 1 line.